### ISSUE TITLE
CI: Suite-k8s-1.14.K8sHubbleTest Hubble Observe Test {L3/L4,L7} Flow with hubble-relay (GKE)

### ISSUE THREAD

**Issue by gandro** [cilium/cilium#11720]
### Stacktrace

```
/home/jenkins/workspace/Cilium-PR-K8s-GKE/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:471
hubble observe query timed out on "Exitcode: 1 \nStdout:\n \t \nStderr:\n \t time=\"2020-05-27T12:31:43Z\" level=fatal msg=\"main program execution encountered a fatal error\" error=\"failed to dial grpc: context deadline exceeded\"\n\t command terminated with exit code 1\n\t \n"
Expected
    <*errors.errorString | 0xc0003422f0>: {
        s: "Expected string \"L3_L4\" is not in the filter output of \"{$.Type}\": 10s timeout expired",
    }
to be nil
/home/jenkins/workspace/Cilium-PR-K8s-GKE/src/github.com/cilium/cilium/test/k8sT/hubble.go:191
```

From the logs
```
time="2020-05-27T12:31:43Z" level=error msg="Error executing command '/bin/bash bash -c kubectl exec -n cilium hubble-cli-m8wnp -- hubble observe --follow --output=json --server 10.3.251.219:80 --last 1 --type trace --from-pod 202005271229k8shubbletesthubbleobservetestl3l4flow/app2-b58dc9b5c-ppkxm --to-namespace 202005271229k8shubbletesthubbleobservetestl3l4flow --to-label id=app1,zgroup=testapp --to-port 80'" error="exit status 1"
cmd: "kubectl exec -n cilium hubble-cli-m8wnp -- hubble observe --follow --output=json --server 10.3.251.219:80 --last 1 --type trace --from-pod 202005271229k8shubbletesthubbleobservetestl3l4flow/app2-b58dc9b5c-ppkxm --to-namespace 202005271229k8shubbletesthubbleobservetestl3l4flow --to-label id=app1,zgroup=testapp --to-port 80" exitCode: 1 duration: 5.89349679s stdout:

stderr:
time="2020-05-27T12:31:43Z" level=fatal msg="main program execution encountered a fatal error" error="failed to dial grpc: context deadline exceeded"
command terminated with exit code 1
```

~~The hubble relay IP `10.3.251.219` does not match the output in `kubectl get pods -o wide --all-namespaces`~~ Edit: This is the pod, not the service.
```
	 cilium                                               hubble-relay-7fd769b99d-8zr2g                               1/1     Running   0          4m15s   10.0.24.161   gke-cilium-ci-18-cilium-ci-18-48cc9c9e-wjwb   <none>           <none>
``` 

https://jenkins.cilium.io/view/PR/job/Cilium-PR-K8s-GKE/1520/
[b5293076_K8sHubbleTest_Hubble_Observe_Test_L3-L4_Flow_with_hubble-relay.zip](https://github.com/cilium/cilium/files/4689611/b5293076_K8sHubbleTest_Hubble_Observe_Test_L3-L4_Flow_with_hubble-relay.zip)

**Comment by gandro** [comment]
api-resource-services.txt contians a hubble-relay service which is 6 days old?

```
cilium                                               hubble-relay           ClusterIP   10.3.251.219   <none>        80/TCP              6d14h   k8s-app=hubble-relay
```

**Comment by gandro** [comment]
Closing as stale. This likely has been fixed with #11721
